Library/Formula/gcc.rb

```diff
@@ -21,9 +21,9 @@ def osmajor
 
   desc "GNU compiler collection"
   homepage "https://gcc.gnu.org"
-  url "http://ftpmirror.gnu.org/gcc/gcc-5.2.0/gcc-5.2.0.tar.bz2"
-  mirror "https://ftp.gnu.org/gnu/gcc/gcc-5.2.0/gcc-5.2.0.tar.bz2"
-  sha256 "5f835b04b5f7dd4f4d2dc96190ec1621b8d89f2dc6f638f9f8bc1b1014ba8cad"
+  url "http://gnu.mirror.iweb.com/gcc/gcc-5.3.0/gcc-5.3.0.tar.bz2"
```

Comment:
`ftpmirror.gnu.org` resolves to the nearest mirror, right?  It should probably stay.